frontend/src/components/Board/BuyParts/index.module.scss

```diff
@@ -1,8 +1,10 @@
 @import '../../breakpoint.scss';
 
 .notSelectable {
+  -webkit-user-select: none;
   user-select: none;
   cursor: default;
+  margin-left: 5px;
```

Comment:
Is the -webkit prefix one needed?

Why are we mixing margin concerns with notSelectable?